lints.toml

```diff
@@ -1,5 +1,5 @@
 #
-# For all clippy lints please visit: https://rust-lang.github.io/rust-clippy/master/
+# For all clippy lints please visit: https://rust_lang.github.io/rust_clippy/master/
```

Comment:
Wrong change, new URL doesn't work.

Reply:
Yeah, thanks :)
This was just `sed 's/-/_/g'`